src/parser/character/token.js

```diff
@@ -64,12 +64,13 @@ export function getToken(data) {
   try {
     const versionCompare = utils.versionCompare(game.system.data.version, "1.2.0");
 
-    if (versionCompare >= 0) {
-      return getTokenSensesNew(data);
-    } else {
-      return getTokenSensesOld(data);
-    }
+    const token = (versionCompare >= 0)
+      ? getTokenSensesNew(data)
+      : getTokenSensesOld(data);
 
+    return game.settings.get("ddb-importer", "character-update-policy-name")
+      ? token
+      : { ...token, name: undefined };
```

Comment:
Again, can we move this outside of the parser, so it replaces the name/ removes it before doing the character update.